lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -1319,26 +1319,35 @@ sub lookupFontinfo {
   #  return LookupValue(($defn ? ToString($defn) : ToString($token)) . '_fontinfo'); }
   return LookupValue('fontinfo_' . ($defn ? $defn->getCSName : ToString($token))); }
 
-# Need to handle "at" too!!!
+# This should eventually actually load the font metrics,
+# and tie-in to the FontMetrics data used by Font.
 DefPrimitive('\font Token SkipMatch:= SkipSpaces TeXFileName', sub {
     my ($stomach, $cs, $name) = @_;
     my $gullet = $stomach->getGullet;
     $name = ToString($name);
+    my ($at, $scaled);
+    if ($gullet->readKeyword('at'))     { $at     = $gullet->readDimension; }
+    if ($gullet->readKeyword('scaled')) { $scaled = $gullet->readNumber; }
     my %props = LaTeXML::Common::Font::decodeFontname($name,
-      ($gullet->readKeyword('at')
-        ? $gullet->readDimension->ptValue : undef),
-      ($gullet->readKeyword('scaled')
-        ? $gullet->readNumber->valueOf / 1000 : undef));
+      $at && $at->ptValue, $scaled && $scaled->valueOf / 1000);
     if (!keys %props) {    # Failed?
       Info('unexpected', $name, $stomach, "Unrecognized font name '$name'",
         "Font switch macro " . ToString($cs) . " will have no effect"); }
     else {
       $props{fontname} = $name; }
+    my $f = ($at ? $at->divide(Dimension('1em'))->valueOf
+      : ($scaled ? $scaled->valueOf / 1000
+        : 1));
+    my $fontinfo = {%props};
+    $$fontinfo{data} = [map { $_->multiply($f); }
+        Dimension(0), Dimension('0.5em'), Dimension(0),
+      Dimension(0), Dimension('1ex'), Dimension('1em')];
     $gullet->skipSpaces;
-    AssignValue('fontinfo_' . ToString($cs) => {%props});
+    # Store the font info & metrics
+    AssignValue('fontinfo_' . ToString($cs) => $fontinfo);
+    # The font $cs should select the font
     DefPrimitiveI($cs, undef, undef, font => {%props});
```

Comment:
Ah, my comment from line 1341 applies better here at line 1349, where we truly have the last use of `%props`.

Reply:
Indeed, I'm just not in the `\%` habit. Patch coming.